src/components/AddScreen/AddScreen.js

```diff
@@ -47,12 +47,11 @@ export default class AddScreen extends React.Component {
                 category: this.state.categorySelectedType,
                 done: false,
                 status: "todo",
-                date: date
+                creationDate: date
```

Comment:
do we need to update our mock data too? if so, we should create an issue for it.